env_bootstrap/requirements.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
I'm a little confused about the env_bootstrap/requirements.sh vs. the env_bootstrap/requirements.txt does one make the other obsolete?

Reply:
Up to you I guess. Have not tested .sh version on osx. 
Would need to also keep .txt if you wanted to stick with that install approach.

Reply:
yeah I think leaving the requirements.txt at root makes sense, but maybe you can remove the env_bootstrap/requirements.txt version?

Reply:
Right. yep sounds good.